### PR TITLE
Add missing get columns in aws_ssm_association . Closes #354

### LINKED ISSUE #354: Add missing get columns in aws_ssm_association
**Is your feature request related to a problem? Please describe.**
Add missing get columns in aws_ssm_association

**Describe the solution you'd like**

**Describe alternatives you've considered**

**Additional context**

### PR DESCRIPTION
**Pull request by bigdatasourav**
# Integration test logs
<details>
  <summary>Logs</summary>
  
```
SETUP: tests/aws_ssm_association []

PRETEST: tests/aws_ssm_association

TEST: tests/aws_ssm_association
Running terraform
aws_vpc.my_vpc: Creating...
aws_ssm_document.test_resource: Creating...
aws_ssm_document.test_resource: Creation complete after 5s [id=turbottest93654]
aws_vpc.my_vpc: Still creating... [10s elapsed]
aws_vpc.my_vpc: Creation complete after 13s [id=vpc-0539a3f6610400e30]
aws_internet_gateway.igw: Creating...
aws_security_group.security_group: Creating...
aws_internet_gateway.igw: Creation complete after 6s [id=igw-04dae96dd10777455]
aws_subnet.my_subnet: Creating...
aws_security_group.security_group: Creation complete after 8s [id=sg-0b28b2b47827fb12a]
aws_subnet.my_subnet: Creation complete after 5s [id=subnet-0bb978300ffde52f5]
aws_instance.instance: Creating...
aws_instance.instance: Still creating... [10s elapsed]
aws_instance.instance: Still creating... [20s elapsed]
aws_instance.instance: Still creating... [30s elapsed]
aws_instance.instance: Creation complete after 35s [id=i-0db2f4576557718b5]
aws_ssm_association.named_test_resource: Creating...
aws_ssm_association.named_test_resource: Creation complete after 3s [id=cb0eb51e-63c3-46a8-831e-2b52675f352e]

Warning: Deprecated Resource

The null_data_source was historically used to construct intermediate values to
re-use elsewhere in configuration, the same can now be achieved using locals


Apply complete! Resources: 7 added, 0 changed, 0 destroyed.

Outputs:

account_id = "013122550996"
aws_partition = "aws"
aws_region = "us-east-1"
instance_id = "i-0db2f4576557718b5"
resource_aka = "arn:aws:ssm:us-east-1:013122550996:association/cb0eb51e-63c3-46a8-831e-2b52675f352e"
resource_id = "cb0eb51e-63c3-46a8-831e-2b52675f352e"
resource_name = "turbottest93654"

Running SQL query: test-get-query.sql
[
  {
    "account_id": "013122550996",
    "association_id": "cb0eb51e-63c3-46a8-831e-2b52675f352e",
    "association_name": "turbottest93654",
    "association_version": "1",
    "document_name": "turbottest93654",
    "document_version": "$DEFAULT",
    "partition": "aws",
    "region": "us-east-1",
    "targets": [
      {
        "Key": "InstanceIds",
        "Values": [
          "i-0db2f4576557718b5"
        ]
      }
    ]
  }
]
✔ PASSED

Running SQL query: test-list-query.sql
[
  {
    "association_id": "cb0eb51e-63c3-46a8-831e-2b52675f352e",
    "association_name": "turbottest93654",
    "association_version": "1",
    "compliance_severity": null,
    "overview": {
      "AssociationStatusAggregatedCount": {},
      "DetailedStatus": "Success",
      "Status": "Success"
    },
    "partition": "aws",
    "region": "us-east-1",
    "schedule_expression": null
  }
]
✔ PASSED

Running SQL query: test-notfound-query.sql
null
✔ PASSED

Running SQL query: test-turbot-query.sql
[
  {
    "account_id": "013122550996",
    "akas": [
      "arn:aws:ssm:us-east-1:013122550996:association/cb0eb51e-63c3-46a8-831e-2b52675f352e"
    ],
    "region": "us-east-1",
    "title": "turbottest93654"
  }
]
✔ PASSED

POSTTEST: tests/aws_ssm_association

TEARDOWN: tests/aws_ssm_association

SUMMARY:

1/1 passed.
```
</details>

# Example query results
<details>
  <summary>Results</summary>
  
```
select
  association_id,
  association_name,
  association_version,
  last_execution_date,
  document_name,
  compliance_severity,
  region
from
  aws_ssm_association;
+--------------------------------------+------------------------------------------------------+---------------------+---------------------+--------------------------------------------------+--------------
| association_id                       | association_name                                     | association_version | last_execution_date | document_name                                    | compliance_se
+--------------------------------------+------------------------------------------------------+---------------------+---------------------+--------------------------------------------------+--------------
| cca1247c-1c4e-46ac-ab8c-d2019fc380f7 | AWS-QuickSetup-SSMHostMgmt-ScanForPatches-3z802      | 1                   | 2021-04-30 06:20:13 | AWS-RunPatchBaselineAssociation                  | <null>       
| 5e0e8560-11c0-45d0-962b-715329d5ab31 | SystemAssociationForSsmAgentUpdate                   | 1                   | 2021-04-29 11:55:59 | AWS-UpdateSSMAgent                               | UNSPECIFIED  
| eca5459d-e02b-4f50-9f45-ef2a947119c3 | AWS-QuickSetup-SSMHostMgmt-UpdateSSMAgent-3z802      | 1                   | 2021-04-30 06:22:41 | AWS-UpdateSSMAgent                               | <null>       
| e92ca012-1a0b-4835-96f1-73907111a199 | AWS-QuickSetup-SSMHostMgmt-AttachIAMToInstance-3z802 | 1                   | 2021-04-30 06:19:04 | AWSQuickSetup-CreateAndAttachIAMToInstance-3z802 | <null>       
| 7f12de90-8049-4290-ab69-13b22ace77ca | AWS-QuickSetup-SSMHostMgmt-EnableExplorer-3z802      | 1                   | 2021-04-30 06:18:32 | AWS-EnableExplorer                               | <null>       
| c00d10d1-24fa-463f-a366-52a98be9d11e | AWS-PatchNowAssociation                              | 2                   | 2021-04-29 12:20:16 | AWS-RunPatchBaseline                             | HIGH         
+--------------------------------------+------------------------------------------------------+---------------------+---------------------+--------------------------------------------------+--------------

```
</details>

### CODE REVIEW

File: aws-test/tests/aws_ssm_association/test-get-query.sql [hunk 1]
```diff
@@ -1,3 +1,3 @@
-select association_id, association_version, association_name, document_name, document_version, targets, partition, region, account_id
+select association_id, association_version, association_name, document_name, document_version, targets,  partition, region, account_id
```
Comment (Subhajit97) [inline aws-test/tests/aws_ssm_association/test-get-query.sql:1]:
```suggestion
select association_id, association_version, association_name, document_name, document_version, targets, partition, region, account_id
```

File: aws/table_aws_ssm_association.go [hunk 2]
```diff
@@ -72,29 +90,77 @@ func tableAwsSSMAssociation(_ context.Context) *plugin.Table {
...
+				Description: "A description of the parameters for a document.",
+				Type:        proto.ColumnType_JSON,
+				Hydrate:     getAwsSSMAssociation,
+			},
+			{
+				Name:        "status",
+				Description: "The association status.",
+				Type:        proto.ColumnType_JSON,
+				Hydrate:     getAwsSSMAssociation,
+			},
```
Comment (Subhajit97) [inline aws/table_aws_ssm_association.go:150]: Move this column at the top, after name

File: aws/table_aws_ssm_association.go [hunk 3]
```diff
@@ -72,29 +90,77 @@ func tableAwsSSMAssociation(_ context.Context) *plugin.Table {
...
 				Description: "A cron expression that specifies a schedule when the association runs.",
 				Hydrate:     getAwsSSMAssociation,
 				Type:        proto.ColumnType_STRING,
 			},
+			{
+				Name:        "max_concurrency",
+				Description: "The maximum number of targets allowed to run the association at the same time.",
+				Hydrate:     getAwsSSMAssociation,
+				Type:        proto.ColumnType_STRING,
+			},
```
Comment (Subhajit97) [inline aws/table_aws_ssm_association.go:115]: What is the type of the values stored in this column?
Can we convert the value to `int`? As per description it seems that it stores a integer value
`The maximum number of targets allowed`

Comment (bigdatasourav) [reply aws/table_aws_ssm_association.go:115]: max_concurrency field can be set with percentage also , example 10%, it will be a string.

**Review by Subhajit97 — APPROVED** [review]
LGTM

**Review by cbruno10 — APPROVED** [review]

**Pull request merged**